content/docs/user-guide/assets/pipeline/asset-dependencies-and-identifiers.md

```diff
@@ -7,16 +7,22 @@ toc: true
 ---
 
 Both source and product assets might have dependencies. Source asset and job dependencies are inputs to the **Asset Pipeline** and asset processing jobs. Product dependencies are outputs from the Asset Pipeline and are used to load and package product assets. Asset identifiers help ensure both types of asset dependencies are met.
+
+## Source dependencies
+
+Source dependencies are a process time concept.
```

Comment:
Perhaps "Build Time" is a better term?

Reply:
Hmm, I think we need to lock in some terminology for the steps in game development. In my mind, Build Time is when you're building the C++ code into the executables.